### DISCUSSION TITLE
Question: Where do files containing SQL-queries belong in the project structure?

### DISCUSSION THREAD

**Discussion by drdebian** [kedro-org/kedro#1294] [Idea]
Hi there,

I'm parsing huge amounts of CSVs using IncrementalDataset into an SQLite-Database. Now I'm going to query that data and since the SQL statements are too long to be put in catalog.yml, I want to put them somewhere as a text file to be read by SQLQueryDataSet.

What do the experts here recommend in terms of folder location for this? Right now I'm using ./project/src/sqlstatements/ for this, but I'm unsure if that's "proper".

**Comment by datajoely** [comment]
It's a good question - especially since we only recently added the ability to do this in https://github.com/kedro-org/kedro/pull/887

We typically put declarative configuration in `conf` and imperative business logic in `src`. I think this is on the boundary between the two. Personally speaking - I like your proposal it makes sense to me, but also interested to see if anyone else form the community has an opinion here.
